README.md

```diff
@@ -27,6 +27,10 @@ operators in production.
   logging and error handling.
 - Minimal set of dependencies.
 
+## Docs
+
+- [Keeping Reconciliation Loops Short](docs/keeping_reconciliation_loops_short.md)
```

Comment:
This is just a first catch. I am not sure if the structure is what we want. When you have a cool idea let me know. Otherwise I am pretty sure this will evolve over time as we go.